libcloud/loadbalancer/drivers/dimensiondata.py

```diff
@@ -636,16 +636,18 @@ def ex_create_virtual_listener(self,
         if listener_ip_address is not None:
             ET.SubElement(create_node_elm, "listenerIpAddress").text = \
                 str(listener_ip_address)
-        ET.SubElement(create_node_elm, "port").text = str(port)
+        if port:
```

Comment:
Are any other falsey-values expected here other than None? if not, please can you keep it consistent with the others and specifically say `if port is not None`